### ISSUE TITLE
AsyncIO RuntimeError: Event loop is closed

### ISSUE THREAD

**Issue by pidi3000** [pidi3000/Music_Badger-Youtube_Feed#22]
This error often occures after running the update uploads function which uses AsyncIO to run multiple web requests.

Something is not right with my implementation.
Problem code area:
https://github.com/pidi3000/Music_Badger-Youtube_Feed/blob/9601cddf455ef062e1def63cda335c2efee3d1ea/music_feed/youtube/_uploads.py#L261

## Logs
```
Exception ignored in: <function _ProactorBasePipeTransport.__del__ at 0x0000021B52E3A8C0>
Traceback (most recent call last):
  File "c:\Python310\lib\asyncio\proactor_events.py", line 116, in __del__
    self.close()
  File "c:\Python310\lib\asyncio\proactor_events.py", line 108, in close
    self._loop.call_soon(self._call_connection_lost, None)
  File "c:\Python310\lib\asyncio\base_events.py", line 750, in call_soon
    self._check_closed()
  File "c:\Python310\lib\asyncio\base_events.py", line 515, in _check_closed
    raise RuntimeError('Event loop is closed')
RuntimeError: Event loop is closed
```

**Comment by pidi3000** [comment]
Possible fix/workaround

https://stackoverflow.com/questions/68123296/asyncio-throws-runtime-error-with-exception-ignored

**Comment by pidi3000** [comment]
fixed in #28 
by changing how async tasks are run
